settings.gradle

```diff
@@ -2,90 +2,96 @@ plugins {
     id("de.fayard.refreshVersions") version "0.40.1"
 }
 
-include ':browser-api'
-include ':vpn'
-include ':vpn-store'
-include ':vpn-api'
-include ':vpn-internal'
-include ':di'
 include ':app'
-include ':statistics'
-include ':common'
-include ':common-ui'
-include ':app-store'
-include ':appbuildconfig-api'
 
-include ':feature-toggles-api'
-include ':feature-toggles-impl'
-
-include ':privacy-config-api'
-include ':privacy-config-impl'
-include ':privacy-config-store'
-
-include ':traces-api'
-include ':traces-impl'
+"feature-toggles".with {
+    include ":$it-api"; project(":$it-api").projectDir = new File("$it/$it-api")
+    include ":$it-impl"; project(":$it-impl").projectDir = new File("$it/$it-impl")
+}
 
-include ':anrs-api'
-include ':anrs-impl'
-include ':anrs-store'
+"privacy-config".with {
+    include ":$it-api"; project(":$it-api").projectDir = new File("$it/$it-api")
+    include ":$it-impl"; project(":$it-impl").projectDir = new File("$it/$it-impl")
+    include ":$it-store"; project(":$it-store").projectDir = new File("$it/$it-store")
+}
 
-include ':remote-messaging-api'
-include ':remote-messaging-impl'
-include ':remote-messaging-store'
+"remote-messaging".with {
+    include ":$it-api"; project(":$it-api").projectDir = new File("$it/$it-api")
+    include ":$it-impl"; project(":$it-impl").projectDir = new File("$it/$it-impl")
+    include ":$it-store"; project(":$it-store").projectDir = new File("$it/$it-store")
+}
 
-include ':macos-api'
-include ':macos-impl'
-include ':macos-store'
+"voice-search".with {
+    include ":$it-api"; project(":$it-api").projectDir = new File("$it/$it-api")
+    include ":$it-impl"; project(":$it-impl").projectDir = new File("$it/$it-impl")
+    include ":$it-store"; project(":$it-store").projectDir = new File("$it/$it-store")
+}
 
-include ':bandwidth-impl'
-include ':bandwidth-store'
+"downloads".with {
+    include ":$it-api"; project(":$it-api").projectDir = new File("$it/$it-api")
```

Comment:
Could this be extracted to a function? I am not a gradle expert but seems like that would reduce the code repetition even more.